src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Linux/LinuxUtilizationProvider.cs

```diff
@@ -58,13 +60,13 @@ internal sealed class LinuxUtilizationProvider : ISnapshotProvider
         var meter = meterFactory.Create("Microsoft.Extensions.Diagnostics.ResourceMonitoring");
 #pragma warning restore CA2000 // Dispose objects before losing scope
 
-        _ = meter.CreateObservableGauge(name: ResourceUtilizationCounters.CpuConsumptionPercentage, observeValue: CpuPercentage);
-        _ = meter.CreateObservableGauge(name: ResourceUtilizationCounters.MemoryConsumptionPercentage, observeValue: MemoryPercentage);
+        _ = meter.CreateObservableGauge(name: ResourceUtilizationInstruments.CpuUtilization, observeValue: CpuUtilization, unit: "1");
+        _ = meter.CreateObservableGauge(name: ResourceUtilizationInstruments.MemoryUtilization, observeValue: MemoryUtilization, unit: "1");
```

Comment:
ITel defines different metrics in the [process namespace](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/process-metrics.md) - `process.memory.usage` and `process.memory.virtual`. If we have strong reasons not to report those and define something for .NET, I'd suggest following [JVM pattern](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/runtime/jvm-metrics.md) and put dotnet-only metrics into `dotnet` namespace.

Reply:
We agreed with @klauco to schedule a meeting to discuss memory metrics `ResourceMonitoring` component